comitup/config.py

```diff
@@ -15,12 +15,23 @@
 import os
 import random
 import shutil
+import re
+import logging
+import subprocess
 
 from comitup import persist
 
+log = logging.getLogger('comitup')
+
 PERSIST_PATH = "/var/lib/comitup/comitup.json"
 CONF_PATH = "/etc/comitup.conf"
 BOOT_CONF_PATH = "/boot/comitup.conf"
+DEFAULT_AP = "comitup-<nnn>"
+
+# ap_name regex patterns
+REGEX_APNAME_FORMAT = r'[a-zA-Z\<\>](\w*)-\<([nMs]+?)\>'  # validate format name-<###>|<hostname>-<###>
+REGEX_APNAME_SPEC = r'(?<=\-<).+?(?=\>)'   # extract ### from name-<###>, exclude <>
```

Comment:
But what if the user wants more than one variable in the name? I think I prefer the more explicit replacement logic in my current code.